modules/auxiliary/gather/behind_cloudflare.rb

```diff
@@ -0,0 +1,445 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Auxiliary::Report
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Behind CloudFlare',
+      'Version'        => '$Release: 1.1.3',
```

Comment:
`Version` key can be removed

Reply:
it is an indication to avoid losing myself. I will delete and add a comment.